0.preprocessing_data/local_preprocessing.sh

```diff
@@ -6,8 +6,15 @@ jupyter nbconvert --to=script --FilesWriter.build_directory=scripts/ notebooks/*
 
 cd scripts/ || exit
 
-python 0.update_file_structure.py --HPC False
-python 1.make_z-stack_images.py
+# python 0.update_file_structure.py --HPC False
+# python 1.make_z-stack_images.py
+
+mapfile -t dirs < <(ls -d ../../data/z-stack_images/*)
+
+for dir in "${dirs[@]}"; do
+    echo "Processing directory: $dir"
+    python 2.z_slice_instensity_normalization.py --input_dir "$dir"
```

Comment:
```suggestion
    python 2.z_slice_intensity_normalization.py --input_dir "$dir"
```
also consider checking the other file names if they are misspelled